fonts/fonts.css

```diff
@@ -0,0 +1,7 @@
+@font-face {
+  font-family: 'Montserrat';
+  src: url('montserrat-regular-webfont.woff2') format('woff2'),
+  url('montserrat-regular-webfont.woff') format('woff');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/max_osipov/pullRequest?prid=5575857)